index.bs

```diff
@@ -456,53 +475,55 @@ that are returned to the caller when a new credential is created, or a new asser
 implementation of {{PublicKeyCredential/[[DiscoverFromExternalSource]](options)}} and
 {{PublicKeyCredential/[[Create]](options)}}.
 
-### `CredentialRequestOptions` Extension ### {#credentialrequestoptions-extension}
+### `CredentialCreationOptions` Extension ### {#credentialcreationoptions-extension}
 
-To support obtaining assertions via {{CredentialsContainer/get()|navigator.credentials.get()}}, this document extends the
-{{CredentialRequestOptions}} dictionary as follows:
+To support registration via {{CredentialsContainer/create()|navigator.credentials.create()}}, this document extends
+the {{CredentialCreationOptions}} dictionary as follows:
 
 <pre class="idl">
-    partial dictionary CredentialRequestOptions {
-        PublicKeyCredentialRequestOptions? publicKey;
+    partial dictionary CredentialCreationOptions {
+        MakeCredentialOptions? publicKey;
```

Comment:
Just noticed this after landing, but shouldn't this be scoped in some way? e.g. PublicKeyMakeCredentialsOptions